addons/OPT/tracker/fn_addMarker.sqf

```diff
@@ -66,10 +66,39 @@ private _markerNamen = format ["OPTMarker%1",GVAR(idMarkerConuter)]
 GVAR(idMarkerConuter) = GVAR(idMarkerConuter) + 1;
 
 //Marker erstellen mit Clib
-[_markerNamen, [_markerdata]] call CFUNC(addMapGraphicsGroup);
+//[_markerNamen, [_markerdata]] call CFUNC(addMapGraphicsGroup);
+//BIS Marker
+_private _marker = createMarkerLocal [_markerNamen, _startPosition];
+_marker setMarkerTypeLocal "mil_triangle";
+_marker setMarkerColorLocal "ColorBlack";
+_marker setMarkerSizeLocal [0.8, 0.8];
+_marker setMarkerAlphaLocal _sichtbarkeit;
+_marker setMarkerTextLocal format["%1",_text];
 	
+private _prioritaetzeit = 0;
+
+switch (_prioritaet) do 
```

Comment:
Die checks in diesem switch-Block müssten auch alle klein geschrieben werden, wenn man meinen Kommentar von oben umsetzen möchte.